project_stage_closed/models/task_stage.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
```

Comment:
@elicoidal AFAIK no guideline has been discussed to enforce this order. I personally prefer and always use the order used here.